### ISSUE TITLE
Test failure with pygments 2.14.0

### ISSUE THREAD

**Issue by mweinelt** [bartTC/django-markup#40]
Hi, with django-markup 1.6.0 and pygments 2.14.0 I see the following test case fail.

```python
___________________ FormatterTestCase.test_rst_with_pygments ___________________

self = <django_markup.tests.test_filter.FormatterTestCase testMethod=test_rst_with_pygments>

    def test_rst_with_pygments(self):
        """
        Having Pygments installed will automatically provide a ``.. code-block``
        directive in reStructredText to highlight code snippets.
        """
        text = open(os.path.join(FILES_DIR, 'rst_with_pygments.txt')).read()
        expected = open(
            os.path.join(FILES_DIR, 'rst_with_pygments_expected.txt')
        ).read()
        result = formatter(text, filter_name='restructuredtext')
>       self.assertEqual(result, expected)
E       AssertionError: '<div[204 chars]span><span class="whitespace">\n</span>  <span[103 chars]v>\n' != '<div[204 chars]span>\n  <span class="keyword">return</span> <[71 chars]v>\n'
E       Diff is 721 characters long. Set self.maxDiff to None to see it.

django_markup/tests/test_filter.py:86: AssertionError
```

**Comment by bartTC** [comment]
Thank you. This has been fixed in v1.7

**Comment by mweinelt** [comment]
Thank you!
